flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyMessage.java

```diff
@@ -52,8 +52,10 @@
 
 /**
  * A simple and generic interface to serialize messages to Netty's buffer space.
+ *
+ * <p>This class must be public as long as we are using Netty 4.0.x. Please check FLINK-7845 for more information.
```

Comment:
actually, Netty 4.0.45 seemed to fix this already

Reply:
yes that's true, according to the cassandra JIRA.

Reply:
added specific Netty version.

Reply:
@zentol: not only according to the cassandra ticket... yesterday we checked ourselves, before we found that